src/UI/ExplorerLink/index.tsx

```diff
@@ -9,27 +9,30 @@ import { WithClassname } from 'UI/types/with-classname';
 export const ExplorerLink = ({
   page,
   text,
-  className = 'dapp-explorer-link'
+  className = 'dapp-explorer-link',
+  children
 }: {
   page: string;
   text?: any;
-} & WithClassname) => {
+} & PropsWithChildren &
+  WithClassname) => {
   const {
     network: { explorerAddress }
   } = useGetNetworkConfig();
 
+  const defaultContent = useMemo(
```

Comment:
I really don't think that we need useMemo here

Reply:
Why not? Is more performant than calling a function on each rerender. And I'd not create a new component just for that piece of code